R/facet-labels.r

```diff
@@ -1,51 +1,118 @@
-#' Label facets with their value.
-#' This is the default labelling scheme.
+#' Default labeller for facets
 #'
-#' @param variable variable name passed in by facetter
-#' @param value variable value passed in by facetter
+#' This labeller uses \code{\link{label_value}()} for single factor
+#' facetting and \code{\link{label_both}()} when multiple factors are
+#' involved.
+#' 
+#' @inheritParams label_value
+#' @inheritParams label_both
+#' @family facet labellers
+#' @export
+label_default <- function(labels, multi_line = TRUE, sep = ": ") {
+  if (length(labels) == 1) {
+    label_value(labels, multi_line)
+  } else {
+    label_both(labels, multi_line)
+  }
+}
+class(label_default) <- c("function", "labeller")
```

Comment:
I think this should be just `class(label_default) <- "labeller")`  (technically function isn't a class, but a type)